### DISCUSSION TITLE
Why i can't enable 2fa authenication with my Vietnamese phone number?

### DISCUSSION THREAD

**Discussion by nam000termux** [community/community#69098] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Github doesn't have a otp for +84 phone number
![image](https://github.com/community/community/assets/77914563/2a07d5e7-3106-45a6-a4f1-f59e09cd839b)

**Comment by melvincwng** [comment]
Hey @nam000termux, at the moment, it seems like Vietnam is not in the list of [Supported countries for SMS authentication](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/countries-where-sms-authentication-is-supported#supported-countries-for-sms-authentication).

As quoted directly from the docs:
> If your country is not on this list, then we aren't currently able to reliably deliver text messages to your country. We update this list periodically.

So, instead of setting up SMS 2FA, you can consider setting up 2FA via a TOTP mobile application instead (alternative method). 

For more information, you can check out this link for a step by step guide on how to do so - [Configuring two-factor authentication using a TOTP mobile app](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication#configuring-two-factor-authentication-using-a-totp-mobile-app)

Hope this clarifies and answers your question 😄
